src/tools/user-agent-explainer/user-agent-result-cards.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div v-if="userAgentInfo">
+    <n-grid :x-gap="12" :y-gap="8" cols="1 s:2" responsive="screen">
+      <n-gi v-for="{ heading, icon, content } in sections" :key="heading">
+        <n-card style="height: 100%">
+          <n-page-header>
+            <template #title>
+              {{ heading }}
+            </template>
+            <template v-if="icon" #avatar>
+              <n-icon size="30" :component="icon" :depth="3" />
+            </template>
+          </n-page-header>
+
+          <br />
+
+          <n-space>
+            <span v-for="{ label, getValue } in content" :key="label">
+              <n-tooltip trigger="hover">
+                <template #trigger>
+                  <n-tag v-if="getValue(userAgentInfo)" type="success" size="large" round>
+                    {{ getValue(userAgentInfo) }}
+                  </n-tag>
+                  <n-text v-else />
```

Comment:
Why this empty n-text node ?

Reply:
The "problem" was that I wanted the texts coming from the undefinedFallback at the bottom of the cards (below the tags). So first the tags, then the undefinedFallback messages. 
But the n-tooltip component expects a body and without a value for a certain field (e.g. device.name) the body would be empty.
Another solution would be to have a small function which first sorts the sections (values at the beginning of the array, undefinedFallbacks at the bottom) and loop over this array. I'll try to add a small utility functions to do the sorting.